tests/test_lua_script.c

```diff
@@ -0,0 +1,49 @@
+#include "manager.h"
+
+extern struct Compartment* loaded_comp;
+
+int
+main(int argc, char** argv)
+{
+    // Initial setup
+    manager_ddc = cheri_ddc_get();
+    setup_intercepts();
+
+    /* Setting up compartment internally */
+    char* file = "./lua_script";
+    struct Compartment* hw_comp = comp_from_elf(file);
+    loaded_comp = hw_comp; // TODO
+    log_new_comp(hw_comp);
+    comp_map(hw_comp);
+
+    /* Compartment execution varialbes */
+    char* comp_fn;
+    int comp_result;
+
+    /* Test `simple_val */
+    comp_fn = "simple_val";
+    /*comp_result = comp_exec(hw_comp, comp_fn, NULL, 0);*/
```

Comment:
Commented code.

Reply:
I think this whole file is now unneeded as I reworked how the testing is done. I've just removed it in b9d2738e0815ceb9c3b1d4b0469015d8a4ed52a9.